recipes/dbus-cxx/2.x.x/conanfile.py

```diff
@@ -0,0 +1,139 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeDeps
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain
+from conan.tools.files import get, replace_in_file, rmdir, copy
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
+
+class DbusCXX(ConanFile):
+    name = "dbus-cxx"
+    license = "LGPL-3.0-only", "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://dbus-cxx.github.io"
+    description = "DBus-cxx provides an object-oriented interface to DBus"
+    topics = "bus", "interprocess", "message"
+    package_type = "shared-library"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "with_glib": [True, False],
+        "with_qt": [True, False],
+        "with_uv": [True, False],
+    }
+    # Default to a relocatable static library with only the default
+    # standalone dispatcher
+    default_options = {
+        "fPIC": True,
+        "with_glib": False,
+        "with_qt": False,
+        "with_uv": False,
+    }
+
+    def validate(self):
+        if self.settings.os == "Windows" and any([
+            self.options.get_safe("with_uv"),
+            self.options.get_safe("with_qt"),
+            self.options.get_safe("with_glib")
+        ]):
+            raise ConanInvalidConfiguration("Using any of these options (with_uv, with_glib, "
+                                            "with_qt) is not working on Windows.")
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 17)
+        # FIXME: Next release will likely be able to use static/shared mode.
+        if self.options.get_safe("with_uv") and not self.dependencies["libuv"].options.shared:
+            raise ConanInvalidConfiguration(f"libuv needs to be shared for "
+                                            f"{self.name}/{self.version}: \"libuv/*:shared=True\"")
+
+    def requirements(self):
+        self.requires("libsigcpp/3.0.7", transitive_headers=True)
+        if self.options.get_safe("with_glib"):
+            self.requires("glib/2.81.0", transitive_headers=True)
+        if self.options.get_safe("with_uv"):
+            self.requires("libuv/[>=1 <2]", transitive_headers=True)
+        if self.options.with_qt:
+            self.requires("qt/[~5.15]", transitive_headers=True)
```

Comment:
At this moment, this generates a conflict:

```
ERROR: Version conflict: Conflict between zstd/1.5.5 and zstd/1.5.7 in the graph.
Conflict originates from libmysqlclient/8.1.0

Run 'conan graph info ... --format=html > graph.html' and open 'graph.html' to inspect the conflict graphically.
```
I've forced it using `self.requires("zstd/[1.5.7]", force=True)` to test this option locally too.

I'll likely open another PR to add `self.requires("zstd/[>=1.5 <1.6]")` or `self.requires("zstd/[~1.5]"` to `libmysqlclient`.